flink-core/src/main/java/org/apache/flink/api/common/io/DelimitedInputFormat.java

```diff
@@ -450,7 +450,7 @@ public FileBaseStatistics getStatistics(BaseStatistics cachedStats) throws IOExc
 		}
 		catch (Throwable t) {
 			if (LOG.isErrorEnabled()) {
-				LOG.error("Unexpected problen while getting the file statistics for file '" + this.filePath + "': "
+				LOG.error("Unexpected problem while getting the file statistics for file '" + this.filePath + "': "
```

Comment:
sorry, should I create a separate PR? (a separate JIRA is definitely overkill for this)